src/xrEngine/main.cpp

```diff
@@ -298,6 +297,7 @@ ENGINE_API int RunApplication()
     else
         Console->Execute("renderer renderer_r1");
 
+    Device.Initialize();
```

Comment:
It seems that if `CheckBenchmark()` will return true, then Device will be unitialized

Reply:
But anyway I just fixed it by moving initialization stages without changing them. Sorry if this was impolite.